architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -560,3 +560,26 @@ parameter is not specified, OpenShift sets it based on the image's tag:
 
 . If the tag is *latest*, OpenShift defaults `*imagePullPolicy*` to `*Always*`.
 . Otherwise, OpenShift defaults `*imagePullPolicy*` to `*IfNotPresent*`.
+
+=== Creating an Image Stream by Manually Pushing an Image
+
+An image stream can be automatically created by manually pushing an image to the internal registry.  To
+push an image to the internal registry you must first
+link:../../install_config/install/docker_registry.html#access-logging-in-to-the-registry[log in to the registry].
+
+====
+----
+$ oc get is
```

Comment:
Instead of `oc get is` I'd give here a `docker tag` command, if the next one is docker push. Let's make it explicit, imho.